src/clusterfuzz/_internal/google_cloud_utils/batch.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2023 Google LLC
```

Comment:
General comment: we should move a lot of these into our local_config yaml files intead.